chapter_08_prettification.asciidoc

```diff
@@ -971,7 +1012,7 @@ INSTALLED_APPS = [
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
-    "lists",
+    "lists.apps.ListsConfig",
```

Comment:
Btw my `INSTALLED_APPS` have single quote entries, why are these double quotes? Maybe some auto-formatter?

Reply:
i ran black on *everything* at some point.  and i think django does too now, so everything in settings.py has double quotes these days...

(also i change to just using "lists" rather than `lists.apps.ListConfig` because the latter just seems needlesscly complicated.